src/react-components/icons/svg/arrow-xsmall.svg

```diff
@@ -1 +1,3 @@
-<svg width="14" height="9" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.984 7.578L.424 1.514 1.9.163l5.22 5.695L12.341.163l1.475 1.351-5.56 6.064a1.542 1.542 0 01-2.272 0zm1.474-1.351z"/></svg>
+<svg width="8" height="12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we actually revert this for the time being? I think I'd like to do icons all at once. (Unless this is just a save-as with a code formatter?)

Reply:
yeah I'll revert it